index.html

```diff
@@ -24,19 +24,30 @@
 <body>
    <section class="body">
     <section class="sect">
+
+      
```

Comment:
- [ ] OPTIONAL
Please do not leave any empty lines which do not serve any purpose in your code, to enhance a cleaner code

Reply:
okay thanks